configuration/src/chains/ethereum/submitter/mod.rs

```diff
@@ -39,9 +39,10 @@ pub enum TxSubmitterConf {
 
 impl From<crate::TxSubmitterConf> for TxSubmitterConf {
```

Comment:
I think it would be better to import this with an alias vs qualifying it with `crate::` -- the name overlap is pretty confusing at first glance